node-api-media.md

```diff
@@ -0,0 +1,16 @@
+# Node-API Media
+
+In addition to written documentation and code samples, videos compliment the
+available resources on learning Node-API (formerly N-API) methodologies and
+principles. The following table contains media links to conferences and
+workshops discussing Node-API features, best practices, and general design
+principles for creating native Node.js modules using Node-API.
+
+|Link|Title|
```

Comment:
Maybe you should invert the `Link` and `Title` and releative content.

Reply:
Addressed!